### PR TITLE
Increase int field to long

### PR DESCRIPTION
**Pull request by snappet-panksy**
Pipedrive allows you to enter numbers way bigger than int32 in Custom Int fields. This causes errors in the API when it happens.

### CODE REVIEW

**Comment by snappet-panksy** [comment]
Actually I think this is a bad idea what I did as its a breaking change., but i'm not sure what the actual solution should be.

**Comment by DavidRouyer** [comment]
I'll replace it with a `LongCustomField` type. I don't worry too much for the breaking changes as I don't have a 1.0.0 stable release yet.

File: src/Pipedrive.net/Models/Common/CustomFields/IntCustomField.cs [hunk 1]
```diff
@@ -3,9 +3,9 @@
     // int
     public class IntCustomField : ICustomField
```
Comment (robertmaidlaAW) [inline src/Pipedrive.net/Models/Common/CustomFields/IntCustomField.cs:4]: The class should also be renamed to 'LongCustomField'

**Comment by robertmaidlaAW** [comment]
An additional change is required as well with this modification in `CustomFieldConverter` class:
`customFields.Add(property.Name, new IntCustomField((int)property.Value));` should be set to `customFields.Add(property.Name, new LongCustomField((long)property.Value));`

**Comment by DavidRouyer** [comment]
Fixed in 2c50bfd9ac39f9721664415cfa8eb27c81c575ca

**Pull request closed without merging**